packages/calcite-tokens/tools/token-transformer/parse/expandComposites.test.ts

```diff
@@ -0,0 +1,144 @@
+const mockCorrectTypeCompoundToken = {
+  core: {
+    1: {
+      type: "sizing",
+      value: "10",
+    },
+  },
+  compound: {
+    value: {
+      fontFamily: "$core.font.font-family.primary",
+      fontWeight: "$core.font.font-weight.light",
+      lineHeight: "$core.font.line-height.fixed.0",
+      fontSize: "$core.font.font-size.0",
+      letterSpacing: "$core.font.letter-spacing.normal",
+      paragraphSpacing: "$core.font.paragraph-spacing.normal",
+      textDecoration: "$core.font.text-decoration.none",
+      textCase: "$core.font.text-case.none",
+    },
+    type: "typography",
+  },
+};
+const mockTransformedCompoundTokens = {
+  core: {
+    1: {
+      type: "sizing",
+      value: "10",
+    },
+  },
+  compound: {
+    "font-family": {
+      value: "$core.font.font-family.primary",
+      type: "font-family",
+    },
+    "font-weight": {
+      value: "$core.font.font-weight.light",
+      type: "font-weights",
+    },
+    "line-height": {
+      value: "$core.font.line-height.fixed.0",
+      type: "line-heights",
+    },
+    "font-size": {
+      value: "$core.font.font-size.0",
+      type: "font-size",
+    },
+    "letter-spacing": {
+      value: "$core.font.letter-spacing.normal",
+      type: "letter-spacing",
+    },
+    "paragraph-spacing": {
+      value: "$core.font.paragraph-spacing.normal",
+      type: "paragraph-spacing",
+    },
+    "text-decoration": {
+      value: "$core.font.text-decoration.none",
+      type: "font-style",
+    },
+    "text-case": {
+      value: "$core.font.text-case.none",
+      type: "text-case",
+    },
+  },
+};
+
+const handleTokenStudioVariables = jest.fn((token) => (token.includes("$") ? `{${token.replace(/\$/g, "")}}` : token));
+const convertTokenToStyleDictionaryFormat = jest.fn(() => handleTokenStudioVariables);
+const shouldExpand = jest.fn().mockReturnValue(true);
+const expandToken = jest.fn().mockReturnValue(mockTransformedCompoundTokens);
+
+jest.mock("../utils/compositeTokens.js", () => {
+  const originalModule = jest.requireActual("../utils/compositeTokens.js");
+  return {
+    __esModule: false,
+    ...originalModule,
+    shouldExpand,
+    expandToken,
+  };
+});
+
+jest.mock("../utils/convertTokenToStyleDictionaryFormat.js", () => {
+  const originalModule = jest.requireActual("../utils/convertTokenToStyleDictionaryFormat.js");
+  return {
+    __esModule: false,
+    ...originalModule,
+    convertTokenToStyleDictionaryFormat,
+  };
+});
+
+import * as expandComposites from "./expandComposites";
```

Comment:
no. Jest mocks must come first or it doesn't work.

Reply:
> Jest mocks must come first or it doesn't work

IIRC the mocks are hoisted so we should be able to put the import statements at the top. If I'm wrong about that, I think we could put the mocks in a `__mocks__` directory right? It's been a while since I used Jest outside of Stencil.